src/camera/camera_target.py

```diff
@@ -26,6 +27,11 @@ def __post_init__(self):
     def __iter__(self):
         return iter(self._pos)
 
+    @property
+    def animation_name(self):
+        """The name of particular animation sequence."""
```

Comment:
Slight nitpicking, but I think this docstring could be made more explicit

Reply:
fair point, I can extend it